Create Events-Guidelines.md (#296)
cilium/cilium.io: Events-Guidelines.md

## Introduction

This document highlights the events page guidelines for use on the Cilium project's website.

## Objectives

> The objectives of adding events to the Cilium project's website is to:

Reviewer: ```suggestion
The objectives of adding events to the Cilium project's website is to raise awareness of physical and virtual events that:
```

- Share project news, roadmap updates, news releases, and user stories
- Educate on project updates
- Share information on how-tos, demos, etc that are valuable to the community
- Promote awareness for the Cilium project to a broader audience

> - Drive engagement with the community
> - Drive engagement and participation in the project

Reviewer: ```suggestion
- Drive community engagement and participation in the project
```

Wasn't sure these are really two different things, wdyt of this merge?

## Guidelines

For all Cilium events, we remain a neutral project, independent of any company. All events that want to be added to the website need to abide by the following guidelines:

> - Adhere to the project's (or their own) code of conduct and respect user privacy and security

Reviewer: ```suggestion
- Adhere to the project's (or similar) code of conduct and respect user privacy and security
```

- The event must not be about a vendor product
- Activities should be open to the public to benefit the ecosystem as a whole

> - Events should focus primarily on Cilium and its use or updates to the project

Reviewer: ```suggestion
- Events should focus primarily on Cilium
```
I think we can be broad here - anything about Cilium can be up for consideration. For example if we did a "meet the maintainers" session that wouldn't strictly be about Cilium's use or updates, but would still be relevant
